### ISSUE TITLE
Provide an interface to turndown that has the ability to choose how markdown will be rendered

### ISSUE THREAD

**Issue by anishmittal2020** [enrico-kaack/markdown-clipper#13]
Provide an interface to turndown that has the ability to choose how markdown will be rendered. 

For example whether the header should be renderd as Setext or atx. Whether list uses a `-` or a `*`.

Many users have a particular preference, and they want to use the same style everywhere.

**Comment by enrico-kaack** [comment]
Good idea. I think it would be best to provide a settings page with this options. Maybe someone wants to work on this one, i am not actively developing new features for this extension at the moment.

**Comment by ludovicc** [comment]
foambubble/foam requires markdown using a particular format: `-` for lists, 2 spaces for indentation...
